governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Ratification Poll for Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022
+summary: This poll is for accepting the updates to the mandate for Core Unit COM-001.
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp30-governance-communications-core-unit-com-001-mandate-refresh/13197
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-25T16:00:00
+---
+# Ratification Poll for Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, April 11 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c2-SP30 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Governance Communications Core Unit performs a set of programs that focus on MakerDAO’s informational accessibility, stakeholder coordination, and communication practices as a decentralized and public organization.
```

Comment:
I would use this instead:

"MIP39c2-SP30 updates the mandate for Core Unit COM-001. The Governance Communications Core Unit performs a set of programs that focus on MakerDAO's informational accessibility, stakeholder coordination, and communication practices as a decentralized and public organization."

Reply:
Also, fix format